spec/support/email_support/email_objects/new_response_email_html.rb

```diff
@@ -6,78 +6,93 @@ module EmailObjects
       class NewResponseEmailHtml < SitePrism::Page
```

Comment:
All of the email support code has had to change to lookup values from a lookup file depending on the template under test